sermant-backend/src/main/java/io/sermant/backend/server/EventServer.java

```diff
@@ -64,10 +66,15 @@ private EventServer() {
     public void init() {
         eventServer = this;
         if (Objects.requireNonNull(backendConfig.getDatabase()) == DatabaseType.REDIS) {
-            eventServer.daoService = new EventDaoForRedis(backendConfig);
-        } else {
-            eventServer.daoService = new MemoryClientImpl(backendConfig);
+            String[] addressList = backendConfig.getUrl().split(CommonConst.REDIS_CLUSTER_SPLIT);
+            if (addressList.length > 1) {
+                eventServer.daoService = new RedisClusterClientImpl(backendConfig);
+                return;
+            }
+            eventServer.daoService = new RedisClientImpl(backendConfig);
+            return;
```

Comment:
RedisClusterClientImpl already support stand-alone mode

Reply:
RedisClusterClientImpl can be connected only after the Redis cluster configuration is enabled.